flank-scripts/src/main/kotlin/flank/scripts/ops/firebase/SaveServiceAccount.kt

```diff
@@ -0,0 +1,29 @@
+package flank.scripts.ops.firebase
+
+import flank.common.defaultCredentialPath
+import flank.common.downloadFile
+import java.io.File
+import java.nio.file.Path
+
+fun saveServiceAccount(
+    serviceAccount: String,
+    serviceAccountPath: Path = defaultCredentialPath.toAbsolutePath()
+) = when {
+    serviceAccount.startsWith("http", true) -> downloadFile(serviceAccount, serviceAccountPath)
```

Comment:
Is it possible to pass service acc via HTTP request? (just curious)

Reply:
if you have hosted in somewhere, I do not see a problem 😄

Reply:
Never said it's a problem, I was curious about use case

Reply:
I mean that there is no problem to host a service account somewhere and read it later on